tools/java_api/java_test/TestHelper.java

```diff
@@ -0,0 +1,72 @@
+package tools.java_api.java_test;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.File;
+
+
+import tools.java_api.*;
+
+public class TestHelper {
+    private static KuzuDatabase db;
+    private static KuzuConnection conn;
+
+    private static void deleteFolder(File folder) {
+        if (folder.exists()) {
+            File[] files = folder.listFiles();
+            if (files != null) {
+                for (File file : files) {
+                    if (file.isDirectory()) {
+                        deleteFolder(file);
+                    } else {
+                        file.delete();
+                    }
+                }
+            }
+            folder.delete();
+        }
+    }
+
+    public static void cleanup() {
+        String folderPath = "java_api_test_db";
```

Comment:
Let's not use a fixed path for testing. Try to create a temp directory with `Files.createTempDirectory`(https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createTempDirectory-java.lang.String-java.nio.file.attribute.FileAttribute...-)